electron/legendary/games.ts

```diff
@@ -185,22 +185,37 @@ class LegendaryGame extends Game {
   public async update() {
     this.state.status = 'updating'
     const { maxWorkers } = (await GlobalConfig.get().getSettings())
-    const workers = maxWorkers === 0 ? '' : `--max-workers ${maxWorkers}`
+    const workers = maxWorkers === 0 ? '' : ` --max-workers ${maxWorkers}`
     const logPath = `"${heroicGamesConfigPath}${this.appName}.log"`
     const writeLog = isWindows ? `2>&1 > ${logPath}` : `|& tee ${logPath}`
-    const command = `${legendaryBin} update ${this.appName} ${workers} -y ${writeLog}`
-
-    try {
-      return await execAsync(command, execOptions).then((v) => {
-        this.state.status = 'done'
-        return v
+    const command = `update ${this.appName}${workers} -y ${writeLog}`.split(' ')
```

Comment:
```suggestion
    const command = `update ${this.appName} ${workers} -y ${writeLog}`.split(' ')
```

Reply:
It was like this before but the thing is that it creates an empty space when the config is not set.

Reply:
Maybe use then something like `join` instead of `$` syntax

Reply:
How can I use join if I need to pass variables?
I think its completely readable. Its only one space that I added there because the output command is putting two spaces there.
Not that it was breaking anything, it was working before and its working now.